### PR TITLE
add update token method

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

**Comment by MixailE** [comment]
[Add re-issuing method for ketl-notification-backend](https://trello.com/c/Sgy9j33G/3834-add-re-issuing-method-for-ketl-notification-backend)

File: src/controllers/token.ts [hunk 1]
```diff
@@ -22,7 +22,29 @@ export default class TokenController {
       return { success: true }
     } catch (e) {
       console.error(e)
-      return ctx.throw(badRequest('Invalid signature'))
+      return ctx.throw(internal(`Can't save token`))
```
Comment (T-Damer) [inline src/controllers/token.ts:25]: Please use common `''`

File: src/controllers/token.ts [hunk 2]
```diff
@@ -22,7 +22,29 @@ export default class TokenController {
...
+      })
+      if (!previousToken) return ctx.throw(badRequest("Can't find the token"))
+
+      previousToken.token = token
+      await previousToken.save()
+
+      return { success: true }
+    } catch (e) {
+      console.error(e)
+      return ctx.throw(internal(`Can't update token`))
```
Comment (T-Damer) [inline src/controllers/token.ts:47]: Once again

File: src/controllers/token.ts [hunk 3]
```diff
@@ -36,7 +58,7 @@ export default class TokenController {
       await TokenModel.deleteMany({ token })
       return { success: true }
     } catch {
-      return ctx.throw(badRequest('Invalid signature'))
+      return ctx.throw(internal(`Can't delete token`))
```
Comment (T-Damer) [inline src/controllers/token.ts:61]: Here too

**Review by T-Damer — APPROVED** [review]

**Pull request merged**